packages/js/src/analysis/plugins/replacevar-plugin.js

```diff
@@ -65,19 +65,26 @@ var YoastReplaceVarPlugin = function( app, store ) {
  * @returns {void}
  */
 YoastReplaceVarPlugin.prototype.registerReplacements = function() {
-	this.addReplacement( new ReplaceVar( "%%currentdate%%",     "currentdate" ) );
-	this.addReplacement( new ReplaceVar( "%%currentday%%",      "currentday" ) );
-	this.addReplacement( new ReplaceVar( "%%currentmonth%%",    "currentmonth" ) );
-	this.addReplacement( new ReplaceVar( "%%currenttime%%",     "currenttime" ) );
-	this.addReplacement( new ReplaceVar( "%%currentyear%%",     "currentyear" ) );
-	this.addReplacement( new ReplaceVar( "%%date%%",            "date" ) );
-	this.addReplacement( new ReplaceVar( "%%userid%%",          "userid" ) );
-	this.addReplacement( new ReplaceVar( "%%id%%",              "id" ) );
-	this.addReplacement( new ReplaceVar( "%%page%%",            "page" ) );
-	this.addReplacement( new ReplaceVar( "%%searchphrase%%",    "searchphrase" ) );
-	this.addReplacement( new ReplaceVar( "%%sitedesc%%",        "sitedesc" ) );
-	this.addReplacement( new ReplaceVar( "%%sitename%%",        "sitename" ) );
-	this.addReplacement( new ReplaceVar( "%%category%%",        "category" ) );
+	this.addReplacement( new ReplaceVar( "%%currentdate%%",       "currentdate" ) );
```

Comment:
I suggest putting things in alphabetical order to make it easier to spot changes.

Reply:
what about the other vars? e.g. excerpt?

Reply:
@increddibelly `%%excerpt%%` is a few lines lower in the code (not visible here on GitHub), it's one of the replace vars like `%%focuskw%%` which is spread out over a few lines of code.

Reply:
I'll push a commit for the alphabetical order.